lib/IDE/Refactoring.cpp

```diff
@@ -5309,7 +5304,7 @@ struct CallbackClassifier {
     // comments.
     CurrentBlock->addPossibleCommentLoc(Statement->getStartLoc());
 
-    ThenBlock->addAllBindings(CallbackConditions, DiagEngine);
+    ThenBlock->addAllBindings(CallbackConditions);
     if (DiagEngine.hadAnyError())
       return;
```

Comment:
Ah just noticed I didn't remove this check, will do along with any review feedback.